Tools/CISupport/ews-build/steps.py

```diff
@@ -2766,7 +2766,7 @@ def evaluateCommand(self, cmd):
                                                 UnApplyPatch(),
                                                 RevertPullRequestChanges(),
                                                 ValidateChange(verifyBugClosed=False, addURLs=False),
-                                                CompileWebKitWithoutPatch(retry_build_on_failure=True),
+                                                CompileWebKitWithoutChange(retry_build_on_failure=bool(self.getProperty('patch_id'))),
```

Comment:
This seems incorrect. We do want this retry behavior for PRs as well. This is the case when a builder built the patch fine, but tester was unable to build it. It indicates some configuration issue with the tester bot. So retrying might result in it being tried on different bot.

Reply:
Ditto at other places for retry_build_on_failure

Reply:
I can revert these for this change, this is the only bit that isn't just a rename anyways, but it seems like a retry is unlikely to improve our situation with a pull request. We'll be building on the same commit that we just failed to build on, it seems like we're likely to get into an infinite retry loop.

Reply:
It's similar to how we retry when kill-old-processes fails (Reference: https://bugs.webkit.org/show_bug.cgi?id=203858). Same bot is unlikely to succeed, but the build might be retried on other bots which should succeed. We do send email notification to bot watchers about bot being in bad state.